receivers/discord/discord.go

```diff
@@ -219,66 +217,49 @@ func (d Notifier) SendResolved() bool {
 
 func (d Notifier) constructAttachments(ctx context.Context, alerts []*types.Alert, embedQuota int) []discordAttachment {
 	attachments := make([]discordAttachment, 0, embedQuota)
-	embedsUsed := 0
-	for _, alert := range alerts {
-		// Check if the image limit has been reached at the start of each iteration.
-		if embedsUsed >= embedQuota {
-			d.log.Warn("Discord embed quota reached, not creating more attachments for this notification", "embedQuota", embedQuota)
-			break
-		}
+	_ = images.WithStoredImages(ctx, d.log, d.images,
```

Comment:
Why are we not checking the error returned here?
```suggestion
	err := images.WithStoredImages(ctx, d.log, d.images,
```

Reply:
Ah, this comes from reverting the code to when it used to use `WithStoredImages`, see https://github.com/grafana/alerting/commit/793c67215ba14d6515b93707caeb7e6d9bf82189#diff-f78cd42a7a3a05c1955ac8b712742062386eaaebb3942dfcd59df91982a9b3d4L208.

It never checked the error from what I can tell, not sure why though. Possibly the intention was to use any attachments that were successful if one failed, but I can check the error and make that explicit instead.